src/main/java/com/github/nutomic/controldlna/mediarouter/MediaRouterPlayService.java

```diff
@@ -335,8 +335,10 @@ public void resume() {
 		intent.putExtra(MediaControlIntent.EXTRA_SESSION_ID, mSessionId);
 		mMediaRouter.getSelectedRoute().sendControlRequest(intent, null);
 		mPollingStatus = true;
-		new CreateNotificationTask().execute(mPlaylist.get(mCurrentTrack)
-				.getFirstPropertyValue(DIDLObject.Property.UPNP.ALBUM_ART_URI.class));
+		if (mCurrentTrack >= 0 && mCurrentTrack < mPlaylist.size()) {
```

Comment:
does 
```
if (mCurrentTrack < 0 || mCurrentTrack >= mPlaylist.size()) {
    mCurrentTrack = 0;
}
```
work better here? It seems logical that if you have never played a track (i.e. mCurrentTrack = -1) it should start by playing track 1, if mCurrentTrack is > the playlist length you could play either the last or the first track but not playing anything seems wrong.